pkgs/applications/misc/terminus/default.nix

```diff
@@ -28,16 +88,22 @@ stdenv.mkDerivation rec {
     mkdir -p "$out/bin"
     mv opt "$out/"
     ln -s "$out/opt/Terminus/terminus" "$out/bin/terminus"
-    patchelf --set-interpreter "$(cat $NIX_CC/nix-support/dynamic-linker)" --set-rpath "${libPath}:\$ORIGIN" "$out/opt/Terminus/terminus"
+
+    patchelf --set-interpreter ${stdenv.cc.bintools.dynamicLinker} \
+        "$out/opt/Terminus/terminus"
+
+    wrapProgram $out/bin/terminus \
+        "''${gappsWrapperArgs[@]}" \
+        --prefix XDG_DATA_DIRS : "${gtk3}/share/gsettings-schemas/${gtk3.name}/" \
```

Comment:
just add `gtk3` and `glib` and this will be added to `gappsWrapperArgs` automatically as intended